chapter5-tensorrt-api-basics/5.5-build-model/models/weights/sample_concat.weights

```diff
@@ -0,0 +1,5 @@
+4
```

Comment:
If all these weights can be reproduced from python code, can you remove all these weights?